### PR TITLE
Make diffraction team updates to release notes

### PR DESCRIPTION
**Pull request by DannyHindson**
**Description of work.**

Make a few minor changes to the release notes for features added by the ISIS diffraction team - including adding a screenshot of a 3D plot showing a result from the new CalculateMultipleScattering algorithm

**To test:**

Review doc changes

No linked issue

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Comment by Harrietbrown** [comment]
in the diffraction 6.1 doc the LeanElasticPeak, im not sure if that should be its own subsection but it does not fit in with the rest of the entries.

File: docs/source/release/v6.1.0/framework.rst [hunk 1]
```diff
@@ -5,6 +5,10 @@ Framework Changes
 .. contents:: Table of Contents
    :local:
 
+.. figure:: ../../images/Multiple_Scatter_Intensity.png
```
Comment (Harrietbrown) [inline docs/source/release/v6.1.0/framework.rst:8]: Could you make it clear what entry this image is connected to. Perhaps by making the entry bold.

**Review by Harrietbrown — CHANGES_REQUESTED** [review]

**Comment by DannyHindson** [comment]
> in the diffraction 6.1 doc the LeanElasticPeak, im not sure if that should be its own subsection but it does not fit in with the rest of the entries.

This is something the SNS team have implemented and the release note is from them so I'd rather not change myself.

It's correct it sits under the Single Crystal Diffraction, New Features headings. I think they've created it as a level 3 heading beneath these although maybe that's not totally obvious given the subtle difference in the text size. It's a fairly major change so rather than just giving it a single bullet like the other "new features" they've given it a level 3 subsection to itself. So perhaps OK?

**Comment by Harrietbrown** [comment]
It is just that it breaks up the new features section. but I suppose it can be left as is for not.

**Review by Harrietbrown — APPROVED** [review]
Thanks these changes look very much better. good job.

**Pull request merged**